packages/flutter_tools/lib/src/device.dart

```diff
@@ -269,8 +269,8 @@ abstract class Device {
 
     // Join columns into lines of text
     return table.map((List<String> row) =>
-        indices.map((int i) => row[i].padRight(widths[i])).join(' • ') +
-        ' • ${row.last}');
+        indices.map((int i) => row[i].padRight(widths[i])).join(' - ') +
+        ' - ${row.last}');
```

Comment:
Windows's cmd cannot print Unicode characters reliably. Why is this a net loss? It's the same Information, just a different separator character? 

New: 
```shell
  Nexus 6P - 84B5T15A10012703 - android-arm - Android 7.1.1 (API 25)
```

Old:
```shell
Nexus 6P • 84B5T15A10012703 • android-arm • Android 7.1.1 (API 25)
```